cmd/calendarsync/main.go

```diff
@@ -111,7 +113,7 @@ func Run(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	log.Info("loaded config file", "path", cfg.Path)
+	log.Debug("loaded config file", "path", cfg.Path)
```

Comment:
I'd prefer to keep the config file and loaded source/sink adapter messages as `Info`. My feeling is that it could be useful that the output clearly states which configuration was used and which calendars were involved.